README.md

```diff
@@ -26,105 +26,106 @@
-* Please provide a pull request if you have or want to create a missing theme.
+| Theme                      |       GitHub       |     CodeMirror     |      Jupyter       |
+|----------------------------| :----------------: | :----------------: | :----------------: |
+| Ambiance                   | :heavy_check_mark: | :heavy_check_mark: |                    |
+| Base16 Ocean Dark          |                    | :heavy_check_mark: | :heavy_check_mark: |
+| Chaos                      | :heavy_check_mark: |                    |                    |
+| Clouds Midnight            | :heavy_check_mark: |                    |                    |
+| Cobalt                     | :heavy_check_mark: | :heavy_check_mark: |                    |
+| Dracula                                         | :heavy_check_mark: | :heavy_check_mark: |
+| GitHub Dark                | :heavy_check_mark: |                    | :heavy_check_mark: |
+| Idle Fingers               | :heavy_check_mark: |                    | :heavy_check_mark: |
+| Kr Theme                   | :heavy_check_mark: |                    |                    |
+| Material                   |                    | :heavy_check_mark: |                    |
+| Merbivore                  | :heavy_check_mark: |                    |                    |
+| Merbivore Soft             | :heavy_check_mark: |                    |                    |
+| Mono Industrial            | :heavy_check_mark: |                    |                    |
+| Mono Industrial Clear      | :heavy_check_mark: |                    |                    |
+| Monokai                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Monokai Spacegray Eighties | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Obsidian                   | :heavy_check_mark: |                    | :heavy_check_mark: |
+| One Dark                   | :heavy_check_mark: | :heavy_check_mark: |                    |
+| Pastel on Dark             | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Solarized Dark             | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Terminal                   | :heavy_check_mark: |                    |                    |
+| Tomorrow Night             | :heavy_check_mark: |                    | :heavy_check_mark: |
+| Tomorrow Night Blue        | :heavy_check_mark: |                    | :heavy_check_mark: |
+| Tomorrow Night Bright      | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Tomorrow Night Eighties    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Twilight (**default**)     | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Vibrant Ink                | :heavy_check_mark: | :heavy_check_mark: |                    |
+
```

Comment:
<img title=":_ cry :" src="https://cloud.githubusercontent.com/assets/136959/13512303/638c0a7e-e15f-11e5-83ab-bb898da7d97d.gif">

Reply:
have some :cake:

Reply:
The 🍰is a lie!

Reply:
I thought cake made everyone feel better. :man_shrugging: